containers/ApiKeyContainer.jsx

```diff
@@ -55,6 +55,11 @@ const ApiKeyContainer = function (ComposedApiKey: ReactClass): ReactClass {
           'terms_and_conditions': '1',
           'send_welcome_email': '0',
           'registration_source': 'open.fda.gov'
+        },
+        options: {
+          'contact_url': 'mailto:jack.finch@semanticbits.com',
```

Comment:
You're absolutely right, intended to change that to our mailing list before I pushed. Corrected.